### ISSUE TITLE
The Control Center do not display on right position

### ISSUE THREAD

**Issue by Apkeniqo** [valinet/ExplorerPatcher#2789]
When I use Automatically hide the taskbar  (Win10) , a part of the control center displays under the taskbar and other application such as Twinkle Tray dose, too. But when I turn off the option ,it returns to normal

**Comment by Vipin2003** [comment]
Working Fine in Lastest Stable version. Share Windows Version

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Apkeniqo** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/90621995/c59a07a0-2f95-4fa1-9604-6cff52c0728c)
![image](https://github.com/valinet/ExplorerPatcher/assets/90621995/098edadc-9d07-413f-afbf-49eec31c3c28)
[ExplorerPatcher_22621.2861.62.2.txt](https://github.com/valinet/ExplorerPatcher/files/14196436/ExplorerPatcher_22621.2861.62.2.txt)
And other software I have installed is TranslucentTB

**Comment by Apkeniqo** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/90621995/2e42bca2-12bc-445a-80a0-91994318ec91)
It works alright when I turn off automatically hide the taskbar

**Comment by pyrates999** [comment]
Uninstall TranslucentTB and reboot.  See if it still happens.

**Comment by Apkeniqo** [comment]
> Uninstall TranslucentTB and reboot. See if it still happens.

it still happens...I have tried to use a newly installed system to install ExplorerPatcher , but it still works innormal...

**Comment by pyrates999** [comment]
Ok, please wait for a developer to respond.

**Comment by Vipin2003** [comment]
> > Uninstall TranslucentTB and reboot. See if it still happens.
> 
> it still happens...I have tried to use a newly installed system to install ExplorerPatcher , but it still works innormal...

Uninstall TranslucentTB and EP with any 3rd party uninstaller (iobit ) completely andr reinstall EP with Latest PR. Then, It will work fine. Check Once

**Comment by Apkeniqo** [comment]
> > > Uninstall TranslucentTB and reboot. See if it still happens.
> > 
> > 
> > it still happens...I have tried to use a newly installed system to install ExplorerPatcher , but it still works innormal...
> 
> Uninstall TranslucentTB and EP with any 3rd party uninstaller (iobit ) completely andr reinstall EP with Latest PR. Then, It will work fine. Check Once

I tried a newly installed system without any third party application installed, but it still happend...The problem happens on any releases of EP

**Comment by Vipin2003** [comment]
Seems very strange..Not happening to other users

**Comment by Bambooster1** [comment]
> Seems very strange..Not happening to other users

It is happening to other users.  With the last Windows Insider update, I lost all system tray icons, clock and action center.

**Comment by Apkeniqo** [comment]
> Seems very strange..Not happening to other users

But it happens on any PC I have used. I don't know if it's caused by system language. I'm using simplified Chinese.

**Comment by Bambooster1** [comment]
for me, I’m on the windows insider program and it only happens in the windows 10 taskbar when I use Explorer patcher. the windows 11 shows up just fine but I hate my taskbar at the bottom of the screen but now I have to leave it there otherwise I lose all of the stuff that’s at the bottom.


Sent from the all new AOL app for iOS


On Sunday, February 11, 2024, 21:55, Apkeniqo ***@***.***> wrote:





Seems very strange..Not happening to other users


But it happens on any PC I have used. I don't know if it's caused by system language. I'm using simplified Chinese.

**Comment by Apkeniqo** [comment]
> for me, I’m on the windows insider program and it only happens in the windows 10 taskbar when I use Explorer patcher. the windows 11 shows up just fine but I hate my taskbar at the bottom of the screen but now I have to leave it there otherwise I lose all of the stuff that’s at the bottom. Sent from the all new AOL app for iOS On Sunday, February 11, 2024, 21:55, Apkeniqo ***@***.***> wrote: Seems very strange..Not happening to other users But it happens on any PC I have used. I don't know if it's caused by system language. I'm using simplified Chinese. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>
Yes, it only happens on windows 10 taskbar with automatically hide. I prefer win10's taskbar to win11's. But the bug happens...It's preventing me to use the function anymore!

**Comment by Apkeniqo** [comment]
> for me, I’m on the windows insider program and it only happens in the windows 10 taskbar when I use Explorer patcher. the windows 11 shows up just fine but I hate my taskbar at the bottom of the screen but now I have to leave it there otherwise I lose all of the stuff that’s at the bottom. Sent from the all new AOL app for iOS On Sunday, February 11, 2024, 21:55, Apkeniqo ***@***.***> wrote: Seems very strange..Not happening to other users But it happens on any PC I have used. I don't know if it's caused by system language. I'm using simplified Chinese. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

Yes, it only happens on win10's taskbar with automatically hide option. The bug it's preventing me to use the function anymore...

**Comment by Apkeniqo** [comment]
> for me, I’m on the windows insider program and it only happens in the windows 10 taskbar when I use Explorer patcher. the windows 11 shows up just fine but I hate my taskbar at the bottom of the screen but now I have to leave it there otherwise I lose all of the stuff that’s at the bottom. Sent from the all new AOL app for iOS On Sunday, February 11, 2024, 21:55, Apkeniqo ***@***.***> wrote: Seems very strange..Not happening to other users But it happens on any PC I have used. I don't know if it's caused by system language. I'm using simplified Chinese. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

Yes, it only happens on win10's taskbar with automatically hide option. The bug it's preventing me to use the function anymore...

**Comment by pyrates999** [comment]
@Bambooster1 your request to fix that was declined and closed because it was the same build as canary.

If this is happening on a build for 26000, this should be closed as well.

**Comment by Amrsatrio** [comment]
A fix for this is coming with the reimplemented taskbar that will come later.

**Comment by Apkeniqo** [comment]
> A fix for this is coming with the reimplemented taskbar that will come later.

Thanks for fixing the problem, i have been troubled by it a long time
